### ISSUE TITLE
android setup error

### ISSUE THREAD

**Issue by fukemy** [mpiannucci/react-native-context-menu-view#54]
Hi, i got this error when run android

```
Build file '....node_modules/react-native-context-menu-view/android/build.gradle' line: 113

A problem occurred configuring project ':react-native-context-menu-view'.
> Configuration with name 'compile' not found.

* Try:
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
```

**Comment by mpiannucci** [comment]
Is this a clean build? hard to tell whats wrong from that output unfortunately

**Comment by fukemy** [comment]
here is my step:
```
npm install react-native-context-menu-view --save

Mostly automatic installation
cd ios/
pod install
```

then i open Android studio, hit run -> got this error

**Comment by fukemy** [comment]
any solution here please, your library is very importain for us? Thanks

**Comment by mpiannucci** [comment]
I’ll look into it today, I’m the meantime just use the previous version

**Comment by mpiannucci** [comment]
Okay all fixed, try installing version 1.5.2, let me know if there are any issues!
